UnitTests/KataTests.cs

```diff
@@ -0,0 +1,58 @@
+using UnitTesting;
+using Xunit;
+
+namespace xUnitTests
+{
+    public class KataTests
```

Comment:
Create separate file with Tests for each Kata. One TestFile should cover only one source file with code.